tests/logout.test.js

```diff
@@ -0,0 +1,37 @@
+import LoginPage from '../pages/login.page';
+import LogoutPage from '../pages/logout.page';
+import HomePage from '../pages/home.page';
+import BasePage from '../pages/base.page';
+
+const LOGIN_PAGE = LoginPage;
+const BASE_PAGE = BasePage;
```

Comment:
These are not needed, and BasePage won't be directly used at this level, only for other pages to inherit from it